crates/bevy_ui/src/render/mod.rs

```diff
@@ -382,6 +383,9 @@ pub struct UiBatch {
     pub z: f32,
 }
 
+const TEXTURED_QUAD: u32 = 0;
+const UNTEXTURED_QUAD: u32 = 1;
```

Comment:
In the futures, this could be turned into a `bitflags`, (for example if you want to cram more metadata into the vertex attributes) but I think it should be kept at two `u32` consts for now.

Reply:
Yes, I kept it this basic because the shader will be replaced soon for one that does rounded corners etc.